tests/components/orderentry/Paginate.test.jsx

```diff
@@ -10,13 +10,27 @@ const props = {
   fetchNew: jest.fn(),
 }
 
+const props2 = {
+  totalPage: 5,
+  nextPageUrl: 'http://next=page=some=random=page=and=the=length=20',
+  dispatch: jest.fn(),
+  prevPageUrl: '',
+  patientId: '123eeeeeer',
+  fetchNew: jest.fn(),
+}
+
+const wrapper2 = shallow(<Paginate {...props2} />)
 const wrapper = shallow(<Paginate {...props} />)
 
 describe('Pagination component test-suite', () => {
   it('renders properly', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('Should render pagination component when no previousurl is passed', () => {
```

Comment:
Check the spelling. `previousurl`.

Reply:
I think creating another wrapper here is not necessary. You could update the props using `setProps` in the wrapper at line 23 since all you are changing is the `prevPageUrl`. Also, add a beforeEach function so that you have a clean component for every test. Look at how other tests in this repository have been structured to start this off better.